appveyor.yml

```diff
@@ -0,0 +1,85 @@
+version: 1.0.{build}
+
+services:
+    - iis                 # start IIS
+
+install:
+    - "python --version"
+    - ps: cd iis; .\getModSecurityPkgs.ps1 https://modsecurity.blob.core.windows.net/windows-artifacts
+
+
+before_build:
+    - ps: |
+        # copy the required pkgs from the ModSecurity blob and install them (c++ & ModSecurity)
+        Invoke-WebRequest -Uri https://modsecurity.blob.core.windows.net/windows-artifacts/vcredist_x64.exe -OutFile vcredist_x64.exe
+        Start-Process vcredist_x64.exe -Wait -ArgumentList '/install /passive /norestart'
+        #setup the python virtual env - not a must but still
+        pip install virtualenv
+        virtualenv --version
+        $python_proj_name = "test_crs3"
+        virtualenv $python_proj_name
+        & ".\$python_proj_name\Scripts\activate.ps1"
+        
+
+build_script:
+    - cmd: ./build_release_amd64.bat
+
+before_test:
+    - ps: |
+        $modsec_repo = "C:\projects\ModSecurity"
+        $modsec_conf = "$modsec_repo\iis\wix\modsecurity.conf"
+        $modsec_iis_conf = "$modsec_repo\iis\wix\modsecurity_iis.conf"
+        & {iisreset /stop}  #need to stop iis before copy of files that are used by it
+        #copy the xml file to the schema
+        Copy-Item "$modsec_repo\iis\ModSecurity.xml" -Destination "C:\Windows\System32\inetsrv\config\schema" -Force
```

Comment:
minor detail: %SystemRoot% (aka. $($env:SystemRoot) in powershell) is generally safer than saying c:\windows, because for example on cloud service VM's, windows is on D:

Reply:
Done